manual/photon-dev.xml.template

```diff
@@ -30,6 +30,10 @@
             <Label>DNS Domain</Label>
             <Description>DNS Domain</Description>
         </Property>
+        <Property ovf:key="guestinfo.ntp" ovf:type="string" ovf:userConfigurable="true" ovf:value="pool.ntp.org">
```

Comment:
How did that get in here? Looks like this could cause conflicts with the ntp PR.

Reply:
This is part of the "dev" template that I use to test deployments. Doesn't affect the actual build